packages/action-menu/src/ActionMenu.ts

```diff
@@ -51,6 +51,9 @@ export class ActionMenu extends ObserveSlotPresence(
     @property({ type: String, reflect: true })
     public static: 'white' | 'black' | undefined = undefined;
 
+    @property({ type: Boolean, reflect: true })
```

Comment:
This does not need to `reflect` as it has no effect of the visual delivery of the `<sp-action-menu>` itself.